bin/deploy-vhost

```diff
@@ -438,7 +444,15 @@ sub update_system_config {
 
     # Create apache/nginx configuration stanzas
     my $vhost_file = "virtualhosts.d/$vhost.conf";
-    mugly("$servers_dir/vhosts/single-vhost.conf.ugly", "/etc/apache2/$vhost_file");
+
+    my $temp_file = mktemp("/tmp/deploy-vhost-apache-XXXXXXXX");
+    my $apache_file = "/etc/apache2/$vhost_file";
+    mugly("$servers_dir/vhosts/single-vhost.conf.ugly", $temp_file);
+    if (!-e $apache_file || system("diff", "-u", $apache_file, $temp_file) != 0) {
+        $restart_apache = 1;
+    }
+    copy($temp_file, $apache_file);
```

Comment:
Why copy if not updated? I know it makes no difference, but I would probably put this inside the `if` block.